src/locale-data-context/LocaleDataContext.tsx

```diff
@@ -0,0 +1,174 @@
+import { csvParse, DSVRowAny } from "d3";
+import { rollup } from "d3-array";
+import numeral from "numeral";
+import React from "react";
+
+type LocaleRecord = {
+  county: string;
+  estimatedIncarceratedCases: number;
+  hospitalBeds: number;
+  reportedCases: number;
+  state: string;
```

Comment:
Even though I don't really like it because we use proper state names, should use `stateCode` here to be consistent with how we use this naming elsewhere and so that we don't confuse it with React's concept of `state`?

Reply:
yeah good call out. what i really want to do is go through and change it to `stateName` everywhere but I have not gotten around to that. we can come back to that later though, i'll file an issue